### PR TITLE
du: Add object counts

### PR DESCRIPTION
**Pull request by klauspost**
Collect object count as well.

Example:

image

Without versions:

image

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: cmd/du-main.go [hunk 1]
```diff
@@ -91,16 +91,25 @@ EXAMPLES:
...
 // Structured message depending on the type of console.
 type duMessage struct {
-	Prefix string `json:"prefix"`
-	Size   int64  `json:"size"`
-	Status string `json:"status"`
+	Prefix     string `json:"prefix"`
+	Size       int64  `json:"size"`
+	Objects    int64  `json:"objects"`
+	Status     string `json:"status"`
+	IsVersions bool   `json:"is_versions"`
```
Comment (vadmeste) [inline cmd/du-main.go:98]: This looks like only useful for internal use, we don't have to export this field

**Review by vadmeste — COMMENTED** [review]
One comment

File: cmd/du-main.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/du-main.go:98]: I think it's exported for the same reason how it's used internally. Exposing this is okay I think. @vadmeste 

However we follow JavaScript style for JSON  in mc so it is better tagged as `isVersion`  @klauspost

Comment (klauspost) [reply cmd/du-main.go:98]: @vadmeste It shows what you are looking at, if only presented with the JSON.

@harshavardhana The rest of the world has snake case for JSON, but I will ofc change it, so we at least consistent.

Comment (vadmeste) [reply cmd/du-main.go]: > It shows what you are looking at

AFAIK,  isVersions is set to true if the user passes --versions flag, it doesn't seem it is useful to output this again to the user.

Comment (vadmeste) [reply cmd/du-main.go]: Oh because we already have 'Objects' which can be confusing.

Comment (klauspost) [reply cmd/du-main.go]: @vadmeste The user may just send the output and not the commandline.

**Review by vadmeste — APPROVED** [review]
LGTM

File: cmd/du-main.go (hunk 1 above)
Comment (vadmeste) [reply cmd/du-main.go]: > @vadmeste The user may just send the output and not the commandline.

Agree. In fact we introduced versioning in MinIO after mc du. That's why non breaking change is hard here.

**Pull request merged**